SampleOfferWithoutReward.sol

```diff
@@ -131,6 +131,10 @@ contract SampleOfferWithoutReward {
     // Executing this function before the Offer is signed off by the Client
     // makes no sense as this contract has no money.
     function getDailyPayment() noEther {
+        if (msg.sender != contractor && msg.sender != address(client)) {
```

Comment:
It is good idea.

However, let's consider removing the `getDailyPayment()` call from the `setDailyWithdrawLimit()`. We should not have issues like before (please verify that!), only the Contractor get less money for the period from the last payout. But:
- the DAO can consider that,
- the Contractor will have the time to payout because the voting will take some time.